README.md

```diff
@@ -1 +1,11 @@
-# roboteam_microcontroller6.0
\ No newline at end of file
+# roboteam_microcontroller6.0
+
+## Generating code with STM32CubeMX
+Sometimes changes to the microcontroller are made and you need to generate new code from STM32CubeMX. To keep the *USER CODE* in for example main.c the following steps should be followed.
+
+1. Make your changes in the .ioc file
+2. Move the main.c file of the board you are working to Core/Src and Core/Inc for .c and .h files respectively.
```

Comment:
Where is the main.c usually found?
Moving to core/src and core/inc, does that mean we can move it in any subfolder of src and inc?